js/kanban.js

```diff
@@ -371,7 +371,7 @@
          self.temp_forms = {};
          var columns = $(self.element + " .kanban-column");
          $.each(columns, function(i, column) {
-            var forms = $(column).find('.kanban-add-form');
+            var forms = $(column).find('.kanban-add-form').find('.kanban-bulk-add-form');
```

Comment:
Should it be this ?
```suggestion
            var forms = $(column).find('.kanban-add-form, .kanban-bulk-add-form');
```

Reply:
I have doubt about the comma -> https://stackoverflow.com/a/39565294

Reply:
`$(column).find('.kanban-add-form').find('.kanban-bulk-add-form');` will return all `'.kanban-bulk-add-form'` that are childs of `'.kanban-add-form'`, which correspond to the CSS selector `.kanban-add-form .kanban-bulk-add-form`

`$(column).find('.kanban-add-form, .kanban-bulk-add-form');` will return all `'.kanban-bulk-add-form'` and `'.kanban-add-form'`, which correspond to the CSS selector `.kanban-add-form, .kanban-bulk-add-form`

Reply:
The case with the comma works as intended for me. Chaining finds does not because the `kanban-add-form` class isn't on the `.kanban-bulk-add-form` elements or as a parent.